Hazelcast.Net/Hazelcast.Core/EntryEvent.cs

```diff
@@ -57,22 +108,24 @@ public virtual TValue GetOldValue()
             return _oldValue;
         }
 
-        public override object GetSource()
-        {
-            return Name;
-        }
-
         /// <summary>Returns the value of the entry event</summary>
-        /// <returns>the valueS</returns>
+        /// <returns>the value</returns>
         public virtual TValue GetValue()
         {
             return _value;
         }
 
+        /// <summary>Returns the incoming merging value of the entry event.</summary>
+        /// <returns>merge value</returns>
+        public virtual TValue GetMergingValue()
+        {
+            return _mergingValue;
+        }
+
         public override string ToString()
         {
-            return "EntryEvent {" + GetSource() + "} key=" + GetKey() + ", oldValue=" + GetOldValue() + ", value=" +
-                   GetValue() + ", event=" + GetEventType() + ", by " + Member;
+            return string.Format("EntryEvent{ {0}, key={1}, oldValue={2}, value={3}, mergingValue={4} }",
```

Comment:
order of oldValue and value seems mixed ?

Reply:
done